cluster-provision/centos8/scripts/vm.sh

```diff
@@ -150,6 +150,6 @@ exec qemu-system-x86_64 -enable-kvm -drive format=qcow2,file=${next},if=virtio,c
   -append "$(cat /kernel.args) $(cat /additional.kernel.args) ${KERNEL_ARGS}" \
   -vnc :${n} -cpu host,migratable=no,+invtsc -m ${MEMORY} -smp ${CPU} \
   -serial pty -M q35,accel=kvm,kernel_irqchip=split \
-  -device intel-iommu,intremap=on,caching-mode=on -soundhw hda -soundhw ac97 \
+  -device intel-iommu,intremap=on,caching-mode=on -device intel-hda -device hda-duplex -device AC97 \
```

Comment:
now that `-shoundhw` is gone, `-device AC97` is more opaque; but why do we need audio on kubernetes nodes? Can it be dropped (on another PR?)

Reply:
Sorry I removed these too soon - soundhw is only removed qemu 7.1[1]. Fedora 36 has qemu 6.2. I have added the soundhw flags back in as they are more descriptive of what the device is. These soundcards are used for testing scenarios with the Kubevirt device plugin - https://github.com/kubevirt/kubevirtci/pull/734

[1] https://github.com/qemu/qemu/blob/master/docs/about/removed-features.rst#creating-sound-card-devices-using--soundhw-removed-in-71

Reply:
Just note please that we do have this warning
```
qemu-system-x86_64: warning: '-soundhw ac97' is deprecated, please use '-device AC97' instead
qemu-system-x86_64: warning: '-soundhw hda' is deprecated, please use '-device intel-hda -device hda-duplex' instead
```